ros_ws/src/controller/src/serial.cpp

```diff
@@ -0,0 +1,132 @@
+// System includes
```

Comment:
What is this file for if the `ControllerNode` has its own serial implementation?

Reply:
Had started a separate file toward making this modular. To make things clear, I will remove it from this pr and have a separate branch.